backend/fms_core/tests/test_models.py

```diff
@@ -197,7 +197,7 @@ def test_extracted_sample(self):
         volume_used = Decimal('0.01')
         parent_sample = self.parent_sample
         s = Sample.objects.create(**create_extracted_sample(sample_kind=self.sample_kind_DNA,
-                                                            volume_used=volume_used,
+                                                            volume=volume_used,
```

Comment:
see above comment. the volume_used value should be inserted in the ProcessSample only... This applies to the other similar changes below.